source/Octopus.Tentacle/Diagnostics/Metrics/MapFromConfigMapToEventList.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+
+namespace Octopus.Tentacle.Diagnostics.Metrics
+{
+    public class MapFromConfigMapToEventList
+    {
+        public List<EventRecord> FromConfigMap(List<EventRecord> configMapContent)
+        {
+            return configMapContent;
```

Comment:
I'm guessing this is still WIP so this class is here in case we want to transform the list into a more 'compact' representation for storage?

Reply:
This is a WIP.
And it depends on how much we want to go for a "normalised" form of the data.

Reply:
removed this concept - we store a dictionary of dictionaries as per informal review.